buildscripts/CMakeLists.txt

```diff
@@ -37,16 +37,16 @@ set(MYGUI_VERSION 3.2.2)
 set(MYGUI_HASH SHA256=0a28d7ec8a47993cb68deb48b36331e28f12dd92580b709eaef21d599b67a78f)
 
 # https://github.com/ptitSeb/gl4es
-set(GL4ES_VERSION v1.1.2)
-set(GL4ES_HASH SHA256=0774a1d0500c3a5d4da2d8cf3054ce4780a53cd20fe607024e1ceb38823f195e)
+set(GL4ES_VERSION d8f996bf76196e43adaf01e266b36aba62310e67)
+set(GL4ES_HASH SHA256=c4f0b7bb0b0a6ac897d7afee1451c7982b6fbcbc82d1ceb3e7be1bb5eeff28dc)
 
 # https://github.com/openscenegraph/OpenSceneGraph/releases
 set(OSG_VERSION 3.6.5)
 set(OSG_HASH SHA256=aea196550f02974d6d09291c5d83b51ca6a03b3767e234a8c0e21322927d1e12)
 
 # https://github.com/OpenMW/openmw/commits/master
-set(OPENMW_VERSION dc2c9ee343931810b11b9e543d6361a4ef4bc420)
-set(OPENMW_HASH SHA256=0ee17d980ede9f98e9044b4e6723e1322935fb39f34219f633bd0158185ba972)
+set(OPENMW_VERSION 8c4acc69c484ada1af055d34f888ce5f8e8708bc)
```

Comment:
Until OpenMW 0.46 is released, we'll stick to following the openmw-46 branch (unless it takes them another year or something).

Reply:
it may happen :D i just selected the newest one, which have float/int fixes in shaders, so gl4es can compile them fine. I think this change wont be present in 0.46 stable release. (It was merged a week ago)